python/cudf/cudf/core/column/interval.py

```diff
@@ -71,36 +71,8 @@ def to_arrow(self) -> pa.Array:
             struct_arrow = pa.array([], typ.storage_type)
         return pa.ExtensionArray.from_storage(typ, struct_arrow)
 
-    @classmethod
-    def from_struct_column(
```

Comment:
Is this user-facing / API breaking? Or does the base class provide `from_struct_column` too?